portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,33 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Nikita!");
-  }
+    ArrayList<Comment> comments; 
```

Comment:
List<Comment> comments;

The ArrayList is an implementation detail that doesn't need to be part of the type.